build/rhel.Dockerfile

```diff
@@ -1,10 +1,22 @@
-# Build the manager binary
-FROM registry.redhat.io/rhel8/go-toolset:1.13.4-27 as builder
+# Copyright (c) 2021 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+#
 
-ENV GOPATH=/go/
+# https://access.redhat.com/containers/?tab=tags#/registry.access.redhat.com/devtools/go-toolset-rhel7
+FROM registry.access.redhat.com/devtools/go-toolset-rhel7:1.13.15-4  as builder
+ENV PATH=/opt/rh/go-toolset-1.13/root/usr/bin:${PATH} \
+    GOPATH=/go/
 USER root
 
-WORKDIR /workspace
+RUN go env GOPROXY
```

Comment:
Note this line is optional. I initially included it since builds were failing when `GOPROXY=direct`, so keeping it could point at a reason for builds to fail, but it might be arcane enough so as to not be necessary.

Reply:
I wonder if `go env GOPROXY` means "do the default" which is `go env GOPROXY=https://proxy.golang.org,direct`

Reply:
It just prints the go environment variables that are currently being used. You'd have to explicitly set `export GOPROXY=proxy.golang.org,direct` to actually change the value. 

Some installs set `GOPROXY=direct`, others set `GOPROXY=proxy.golang.org,direct`, and the former case was breaking the build because of how openshift changed their repos.